osgar/drivers/rosmsg.py

```diff
@@ -525,7 +521,7 @@ def slot_raw(self, timestamp, data):
             self.bus.publish('rot', [round(math.degrees(angle)*100) 
                                      for angle in rot])
```

Comment:
My preferred fix would involve removing this part all together and also the related part calculating the angles.

Reply:
OK, but please not now.